features/calendar/assets/js/calendar.js

```diff
@@ -159,7 +160,8 @@ function handleClientLoad(inEvent)
     cal.inEvent = inEvent;
 
     $("#calendar-link").click(function() {
-            calendarLinkHandler();
+        window.open('http://go/pm-slots');
```

Comment:
This url is Etsy-specific so probably shouldn't be in this commit

Reply:
I will remove that and will use from config.

Reply:
Done.